src/containers/NFT/NFTHeader/index.js

```diff
@@ -0,0 +1,246 @@
+    return (
+      <table>
+        <tbody>
+          {minted && (
+            <tr className="row">
+              <td className="col1">Minted</td>
+              <td className="col2">{minted}</td>
+            </tr>
+          )}
+          {domain && (
+            <tr className="row">
+              <td className="col1">{t('domain')}</td>
+              <td className="col2">
+                <a href={`https://${domain}`} target="_blank" rel="noopener noreferrer">
+                  {domain}
+                </a>
+              </td>
+            </tr>
+          )}
+          {emailHash && (
+            <tr className="row">
+              <td className="col1">{t('email_hash')}</td>
+              <td className="col2">
+                {abbrvEmail}
+                <Copy className="copy" text={emailHash} />
+              </td>
+            </tr>
+          )}
+          {NFTTaxon !== undefined && (
+            <tr className="row">
+              <td className="col1">Taxon ID</td>
+              <td className="col2">{NFTTaxon}</td>
+            </tr>
+          )}
+          {uri && (
+            <tr className="row">
+              <td className="col1">URI</td>
+              <td className="col2">
+                {abbrvURI}
+                <Copy className="copy" text={uri} />
+              </td>
+            </tr>
+          )}
+          {transferFee !== undefined && (
+            <tr className="row">
+              <td className="col1">Transfer Fee</td>
+              <td className="col2">{transferFee}</td>
+            </tr>
+          )}
+        </tbody>
+      </table>
+    );
+  }
+
+  renderSettings() {
+    const { data } = this.props;
+    const { flags } = data;
+
+    const burnable = flags && flags.includes('lsfBurnable') ? 'enabled' : 'disabled';
+    const onlyXRP = flags && flags.includes('lsfOnlyXRP') ? 'enabled' : 'disabled';
+    const trustLine = flags && flags.includes('lsfTrustLine') ? 'enabled' : 'disabled';
+    const transferable = flags && flags.includes('lsfTransferable') ? 'enabled' : 'disabled';
+    return (
+      <table>
+        <tbody>
+          <tr className="row">
+            <td className="col1">Burnable</td>
+            <td className="col2">{burnable}</td>
+          </tr>
+          <tr className="row">
+            <td className="col1">Only XRP</td>
+            <td className="col2">{onlyXRP}</td>
+          </tr>
+          <tr className="row">
+            <td className="col1">Trust Line</td>
+            <td className="col2">{trustLine}</td>
+          </tr>
+          <tr className="row">
+            <td className="col1">Transferable</td>
+            <td className="col2">{transferable}</td>
+          </tr>
+        </tbody>
+      </table>
+    );
+  }
+
+  renderHeaderContent() {
+    const { t, data, language, tokenId } = this.props;
+    const { issuer } = data;
+    const abbrvIssuer = issuer && issuer.replace(/(.{23})..+/, '$1...');
```

Comment:
there should always be an issuer

Reply:
also i'm a bit confused as to what this regex is supposed to do. looks like you intended it to truncate characters 24 and beyond and replace them with `...`. This doesn't look right to me if that's the intention. Also, this is simple enough to do without regexes: `issuer.slice(0, 23).concat('...')`.

Reply:
but it does work on paper tho... i am actually just copying it from the existing token page 😂

Reply:
> there should always be an issuer

the check here is to make sure its not undefined, since it will be undefined before the data arrives